my_game.py

```diff
@@ -305,10 +283,11 @@ def __init__(self, width, height):
         # Set up the player info
         self.player_sprite = None
         self.player_score = None
-        self.player_lives = None
+        self.player_lives = 3
         self.player_speed = 0
         self.opposite_angle = 0
         self.max_speed = PLAYER_SPEED
+        self.player_immortality_timer = INVINCIBILITY_SECONDS
 
```

Comment:
This timer should be in the _Player()_ class.

Reply:
deleted it and put INVINCIBILTY_SECONDS in when the player is made in Setup!